tests_common/pytest_plugin.py

```diff
@@ -1243,7 +1246,12 @@ def maker(
         ti.state = state
         ti.external_executor_id = external_executor_id
         ti.map_index = map_index
-
+        if hostname:
+            ti.hostname = hostname
+        if pid:
+            ti.pid = pid
+        if last_heartbeat_at:
+            ti.last_heartbeat_at = last_heartbeat_at
```

Comment:
Nit: since these are nullable columns with defaults anyway
```suggestion
        ti.hostname = hostname
        ti.pid = pid
        ti.last_heartbeat_at = last_heartbeat_at
```

Reply:
Done